portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,96 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random personal fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts = [
+    'I have 2 siblings.',
+    'My favorite color is blue.',
+    'I enjoy outdoor activities.',
+    'I skipped a grade.',
+    'I was born in Massachusetts.',
+  ];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  // Pick a random fact.
+  const fact = facts[Math.floor(Math.random() * facts.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const factContainer = document.getElementById('fact-container');
+  factContainer.innerText = fact;
 }
+
+/**
+ * Cycles backwards to the previous background image
+ */
+function prevBackground() {
+  changeBackground(-1);
+}
+
+/**
+ * Cycles forwards to the next background image
+ */
+function nextBackground() {
+  changeBackground(1);
+}
+
+let curBackground = 0;
+
+/**
+ * Advances the background image by a given offset from the current image 
+ */
+function changeBackground(offset) {
+  // List of background images and their color themes
+  const backgroundImgs = [
```

Comment:
While it's good practice to declare variables where they are used, in this case, since this seems like a configuration option that we might want to update later, I would probably move this outside of the function scope and make it a static variable:

Background.IMAGES = [
 ...
]